packages/api/bin/cli.js

```diff
@@ -1,12 +1,10 @@
-#!/usr/bin/env node
```

Comment:
Removed due to lint error but this may be necessary 👀

Reply:
Re-added here: https://github.com/nasa/cumulus/pull/3354/commits/7abcdf366f3aa62b5c6f0930a6367d64249a1606

My linter was complaining but I think it's just a configuration issue on my end. This passes CI's lint.